src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -452,28 +455,35 @@ - (void)invokeCommandWithEndpointID:(NSNumber *)endpointID
                           clusterID:(NSNumber *)clusterID
                           commandID:(NSNumber *)commandID
                       commandFields:(id)commandFields
-                     expectedValues:(NSArray<NSDictionary<NSString *, id> *> *)expectedValues
-              expectedValueInterval:(NSNumber *)expectedValueInterval
+                     expectedValues:(NSArray<NSDictionary<NSString *, id> *> * _Nullable)expectedValues
+              expectedValueInterval:(NSNumber * _Nullable)expectedValueInterval
```

Comment:
We still need to document the behavior if one of these is nil and the other is not.  Followup for that, please.

Reply:
Okay I think the latest header doc might do it.